backend/cfg/cfgize.ml

```diff
@@ -385,10 +342,9 @@ type block_info =
 let extract_block_info : State.t -> Mach.instruction -> block_info =
  fun state first ->
   let rec loop (instr : Mach.instruction) acc =
-    let return terminator can_raise =
-      let instrs = List.rev acc in
-      let can_raise = can_raise || can_raise_instrs instrs in
-      { instrs; last = instr; terminator; can_raise }
+    let return terminator instrs =
```

Comment:
Why is this `instrs` parameter added?

Reply:
One of the cases modified `acc` before passing it to `return`.

Reply:
Indeed; sorry for the noise.